dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -1743,6 +1743,664 @@
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseResolution"
+        },
+        {
+          "$ref": "#/definitions/QuantityResolution"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "SpeedResolution",
+      "properties": {
+        "unit": {
+          "type": "string",
+          "enum": [
+            "Unspecified",
+            "MeterPerSecond",
+            "KilometerPerHour",
+            "KilometerPerMinute",
+            "KilometerPerSecond",
+            "MilesPerHour",
+            "Knot",
+            "FootPerSecond",
+            "FootPerMinute",
+            "YardPerMinute",
+            "YardPerSecond",
+            "MeterPerMillisecond",
+            "CentimeterPerMillisecond",
+            "KilometerPerMillisecond"
+          ],
+          "x-ms-enum": {
+            "name": "SpeedUnit",
+            "modelAsString": true
+          },
+          "description": "The speed Unit of measurement"
+        }
+      },
+      "required": [
+        "unit"
+      ]
+    },
+    "AreaResolution": {
+      "description": "Represents the area entity resolution model.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseResolution"
+        },
+        {
+          "$ref": "#/definitions/QuantityResolution"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "AreaResolution",
+      "properties": {
+        "unit": {
+          "type": "string",
+          "enum": [
+            "Unspecified",
+            "SquareKilometer",
+            "SquareHectometer",
+            "SquareDecameter",
+            "SquareDecimeter",
+            "SquareMeter",
+            "SquareCentimeter",
+            "SquareMillimeter",
+            "SquareInch",
+            "SquareFoot",
+            "SquareMile",
+            "SquareYard",
+            "Acre"
+          ],
+          "x-ms-enum": {
+            "name": "AreaUnit",
+            "modelAsString": true
+          },
+          "description": "The area Unit of measurement"
+        }
+      },
+      "required": [
+        "unit"
+      ]
+    },
+    "LengthResolution": {
+      "description": "Represents the length entity resolution model.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseResolution"
+        },
+        {
+          "$ref": "#/definitions/QuantityResolution"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "LengthResolution",
+      "properties": {
+        "unit": {
+          "type": "string",
+          "enum": [
+            "Unspecified",
+            "Kilometer",
+            "Hectometer",
+            "Decameter",
+            "Meter",
+            "Decimeter",
+            "Centimeter",
+            "Millimeter",
+            "Micrometer",
+            "Nanometer",
+            "Picometer",
+            "Mile",
+            "Yard",
+            "Inch",
+            "Foot",
+            "LightYear",
+            "Pt"
```

Comment:
should `pt` be "Point"?

Reply:
The service does return Pt so we might need to change this in the next version.